packages/protocol/contracts/L2/DelegateOwner.sol

```diff
@@ -106,6 +108,9 @@ contract DelegateOwner is EssentialContract, IMessageInvocable {
 
         if (_verifyTxId && call.txId != nextTxId++) revert DO_INVALID_TX_ID();
 
+        // By design, the target must be a contract address.
```

Comment:
`msg.value` is supported for a call, so in theory could also just send ETH to an EOA.